test/Telegram.Bot.Tests.Integ/Throwing Exceptions/ThrowingExceptionsTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Telegram.Bot.Exceptions;
+using Telegram.Bot.Tests.Integ.Common;
+using Telegram.Bot.Types;
+using Telegram.Bot.Types.Enums;
+using Xunit;
+
+namespace Telegram.Bot.Tests.Integ.ThrowingExceptions
+{
+    [Collection(CommonConstants.TestCollections.Exceptions)]
+    [TestCaseOrderer(CommonConstants.TestCaseOrderer, CommonConstants.AssemblyName)]
+    public class ThrowingExceptionsTests
+    {
+        public ITelegramBotClient BotClient => _fixture.BotClient;
+
+        private readonly TestsFixture _fixture;
+
+        public ThrowingExceptionsTests(TestsFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        #region 1. Chat not found
```

Comment:
I usually make regions for a group of related tests within a collection. Their excetion order also goes as, for example, 1.1, 1.2, and 1.3(inside region 1).